### ISSUE TITLE
No menu bar with remote X display and 'native' titleBarStyle

### ISSUE THREAD

**Issue by brettbbonner** [microsoft/vscode#100232]
- VSCode Version: 1.46.0
- OS Version: Ubuntu 18.04

Running vscode under Ubuntu 18.04 VM, displaying via forwarded X connection to host system, with window.titleBarStyle set to native as recommended in FAQ.  In this configuration, the vscode window has no menu bar, and it cannot be brought up using Alt key.  'View: Toggle Menu Bar' from command palette causes a brief flash in the upper left.  

Steps to Reproduce:

1. Configure to use window.titleBarStyle: native
2. Start vscode, displaying to remote X server.
3. Observe no title bar; Alt and Toggle Menu Bar have no effect.

Impact:

Can't use menus when displayed remotely.  This was possible previously (regression).

Does this issue occur when all extensions are disabled?: Yes

![image](https://user-images.githubusercontent.com/35403958/84701039-3881c900-af22-11ea-9343-05e09aa9cc6e.png)

**Comment by sbatten** [comment]
@brettbbonner what happens when you use `window.menuBarVisibility` => `visible`?

Also, would you be able to figure out which build it last worked on? You can find older builds here https://code.visualstudio.com/updates/v1_45
https://code.visualstudio.com/updates/v1_44
https://code.visualstudio.com/updates/v1_43

**Comment by brettbbonner** [comment]
window.menuBarVisibility => visible has no effect.

This problem was discovered after I upgraded a development VM that had been on Ubuntu 16.04, to 18.04.  So I did some more experimenting to isolate where it really started.  Tried vscode versions on the Ubuntu 18.04 system, going back past what I was using before, and NONE of them have a working menu bar there.  Curious.  So I went back to a different Ubuntu 16.04 system.  Both the vscode 1.39.2 version that was installed, and 1.46.0 when I updated to that, worked fine there, using the same X windows remote display approach.

So the common thread here is that the vscode menu bar does not work properly on Ubuntu 18.04 with native titlebar and X forwarding.

**Comment by sbatten** [comment]
Thanks for that investigation. Can you confirm inside the VM that menu bar appears correctly? Is your installation using a global menu bar or something other than the traditional style?

**Comment by brettbbonner** [comment]
This VM is a hyper-V hosted one, and the virtual machine "native" window has low resolution and terrible performance so I don't ever use it.  However, I fired it up for testing.  This installation was using defaults which apparently turns on the Mac-style global menu bar.  That did come up for vscode.  I went into System Settings and under Appearance changed "Show the menus for a window" from "In the menu bar" to "In the window's title bar".  Once I logged out and back in, in the native VM window the vscode menus appear when the mouse is over the window title bar.  However, this change did not affect the behavior over the remote X connection.
